src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,69 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driver = new Driver();
+
+        driver.setName("Vasyl");
+        driver.setLicenseNumber("DF12432");
+        driverService.create(driver);
+
+        driver.setName("Sergiy");
+        driver.setLicenseNumber("ER64232");
+        driverService.create(driver);
+
+        driver.setName("Alina");
+        driver.setName("SD078932");
+        driverService.create(driver);
+
+        System.out.println(driverService.get(1L));
```

Comment:
this code always will work correctly?
better use `driverService.get(driver.getId())`